ipn/localapi/localapi.go

```diff
@@ -168,12 +168,23 @@ func validHost(h string) bool {
 	case "", apitype.LocalAPIHost:
```

Comment:
can you do it here?

Reply:
or do they also specify a port?

Reply:
They specify a port: https://github.com/tailscale/tailscale/blob/1ea43d4c87931c74af28209ff1689cd53d291ca1/client/tailscale/localclient.go#L94-L100 and https://github.com/tailscale/corp/blob/c50d5ead9d002d652c0db56449bd720d336c5ed9/xcode/Shared/LocalAPIClient.swift#L307